compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -684,7 +684,7 @@ static void addLowerToLLVMPasses(OpPassManager &passManager,
 
 void buildLLVMCPUCodegenConfigurationPassPipeline(OpPassManager &passManager) {
   {
-    addCommonTargetExecutablePreprocessingPasses(passManager);
+    addCommonTargetExecutablePreprocessingPasses(passManager, false);
```

Comment:
You can add a comment `/*flagName=*/ false`, following the clang-tidy format.

Reply:
Added.